.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts

```diff
@@ -377,9 +378,11 @@ export async function pickTestGroupRunOrder() {
   Fs.writeFileSync('jest_run_order.json', JSON.stringify({ unit, integration }, null, 2));
   bk.uploadArtifacts('jest_run_order.json');
 
-  // write the config for functional steps to an artifact that can be used by the individual functional jobs
-  Fs.writeFileSync('ftr_run_order.json', JSON.stringify(ftrRunOrder, null, 2));
-  bk.uploadArtifacts('ftr_run_order.json');
+  if (ftrConfigsIncluded) {
+    // write the config for functional steps to an artifact that can be used by the individual functional jobs
+    Fs.writeFileSync('ftr_run_order.json', JSON.stringify(ftrRunOrder, null, 2));
+    bk.uploadArtifacts('ftr_run_order.json');
+  }
```

Comment:
I'm a bit confused why we need this logic today? @wayneseymour could you share what is different here b/w jest and functional tests?

Reply:
Well, we dont run the ftr in cc, so we dont need to waste resources uploading and downloading this file.

Reply:
Got it, after reading the whole function. I just confused we do it explicitly for FTR ones :)